fastlane/Fastfile.swift

```diff
@@ -68,4 +68,20 @@ class Fastfile: LaneFile {
         Symbol.downloadFromAppStore(versionNumber: versionNumber, buildNumber: buildNumber)
         Symbol.uploadAppStoreToCrashlytics(versionNumber: versionNumber, buildNumber: buildNumber)
     }
+    
+    // MARK: - Register device
+
+    func registerNewDeviceLane() {
+        let deviceName = prompt(text: "Enter the device name")
```

Comment:
To make it similar with the next line
```suggestion
        let deviceName = prompt(text: "Enter the device name:")
```

Reply:
Good catch! I updated at https://github.com/nimblehq/ios-templates/pull/365/commits/9b582f73aa4a0ceb1c83f52fc7d69fcb26a3b1c1.